### ISSUE TITLE
Missing libappindicator3-1 dependency on Debian Bullseye

### ISSUE THREAD

**Issue by SwissTico** [brimdata/brim#1901]
**Describe the bug**
Impossible to install Brim to Debian Bullseye due to missing dependency libappindicator3-1.
We would recommend in first time to move libappindicator3-1 as "recommended" package instead of "depend", then to replace this old lib for the new "libayatana".

**To Reproduce**

1. Download Brim on a fresh Debian Bullseye installation (NOT upgraded from Buster!)
2. sudo apt install ./Brim*.deb

**Expected behavior**
Use Brim with a very good coffee on hand.
(Finally we got the good coffee, but without Brim ;-)

**Screenshots**
```
sudo apt install ./Downloads/Brim-0.26.0.deb
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
Note, selecting 'brim' instead of './Downloads/Brim-0.26.0.deb'
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 brim : Depends: libappindicator3-1 but it is not installable
E: Unable to correct problems, you have held broken packages.
```

**Desktop Info**
[Please complete the following information]
 - OS: Linux Debian Bullseye
 - Brim Version:0.26.0

**Comment by crashingstatic** [comment]
_**'The accumulated filth of all their updates and broken packages will foam up about their waists and all the users and analysts will look up and shout "Save us!" and I'll whisper "No." ' - The Brimdata Dev Team**_

Here's how you fix the problem:
1) Download the *.deb package
2) Execute the following as root:
```
dpkg-deb -x Brim-*.deb unpack
dpkg-deb --control Brim-*.deb unpack/DEBIAN
sed -i -- 's/libappindicator3-1/libayatana-appindicator3-1/g' unpack/DEBIAN/control
dpkg -b unpack Brim-*.deb
rm -r unpack
apt install ./Brim-*.deb
```

**Comment by facyber** [comment]
@crashingstatic Thank you very much, this helped me make it work. I was struggling for half an hour.

**Comment by philrz** [comment]
It looks like this issue is behind us. Using the currently available Debian Bullseye image that's available (`debian-11.5.0-amd64-DVD-1.iso`, which came out in the time since this issue was opened) the `apt` command line shown in the original repro now successfully installs the `Brim-0.26.0.deb` package. This seems to imply that the original symptom was somehow specific to the older Bullseye that was available at that time.

These screenshot were taken on fresh Debian Bullseye `debian-11.5.0-amd64-DVD-1.iso` installation (not upgraded). The first is with `Brim-0.26.0.deb` package.

![image](https://user-images.githubusercontent.com/5934157/191089228-26af9cb2-9529-48d0-802f-38b064f18134.png)

The same is true for Zui Insiders, which is representative of "current Brim" insofar as the next release will be called Zui and based on the same code.

![image](https://user-images.githubusercontent.com/5934157/191089558-58dbb46e-af79-493f-b4b7-9eb9059073e7.png)

Here's the final proof of both apps having successfully launched after those successful installations.

![image](https://user-images.githubusercontent.com/5934157/191089630-7a42a6a3-b081-4643-9fab-f1692a8ae2dc.png)

If folks are still having difficulties, feel free to open a new issue or come ping us on [public Slack](https://www.brimdata.io/join-slack/).

Closing this issue.
